src/style.css

```
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+  font-style: normal;
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.big-searchbar,
+.small-searchbar {
+  margin-top: 20px;
+}
```

```suggestion
.big-searchbar,
.small-searchbar {
  margin-top: 20px;
}
```
you can add one class name in these blocks